chart/values.yaml

```diff
@@ -49,6 +49,11 @@ networking:
   # If networking is enabled, clusterCIDR must match the PodCIDR subnet your cluster has been configured with.
   # The default "10.244.0.0/16" assumes you're using Flannel with default configuration.
   clusterCIDR: 10.244.0.0/16
+  networkId:
```

Comment:
This can be the name or id, having "Id" in the field name is misleading IMO. What do you think about naming this just "network"?

---

If a user were to use `value: network-name`, how would they set the value so the `valueFrom` is removed?

Reply:
removed the id. Regarding the value... How would someone do it in case of the HCLOUD_TOKEN env var? You'd need to prefer `value` over `valueFrom` using a control sequence. I could add that if you like?